### ISSUE TITLE
External Node Help is not correct

### ISSUE THREAD

**Issue by pbimpisidis** [dfensgmbh/biz.dfch.PS.Appclusive.Client#60]
The external node help including the examples is not the correct one.

for example:
```
DESCRIPTION
 Creates a ExternalNode entry in Appclusive.

You must specify all three parameters 'Name', 'Username' and 'Password'. If the entry already exists no update of the existing entry is performed.
```

**Comment by rufer7** [comment]
@pbimpisidis can you fix it?

**Comment by pbimpisidis** [comment]
I am trying. I got another issue: When I use New-ApcExternalNode without -ExternalType parameter the external node is not created but I get no error.

**Comment by rufer7** [comment]
@pbimpisidis I created another issue for the second bug you mentioned in the last comment

**Comment by rufer7** [comment]
fixed by #66
